collections/datasets/occurharvester.php

```diff
@@ -124,48 +127,46 @@ function openIndPopup(occid){
 	include($SERVER_ROOT.'/includes/header.php');
 	?>
 	<div class='navpath'>
-		<a href='../../index.php'>Home</a> &gt;&gt;
+		<a href='../../index.php'><?php echo $LANG['HOME']; ?></a> &gt;&gt;
 		<?php
 		if(isset($collections_datasets_occurharvesterCrumbs)){
 			echo $collections_datasets_occurharvesterCrumbs;
 		}
 		?>
-		<b>Occurrence Harvester</b>
+		<b><?php echo $LANG['OCCUR_HARV']; ?></b>
 	</div>
 	<!-- This is inner text! -->
 	<div id="innertext">
 		<div style="margin:15px">
-			Scan or type barcode number into field below and then hit enter or tab to add the specimen to the list.
-			Once list is complete, you can enter your catalog number in the text field and then transfer to your collection
-			or file export to a file that can be imported into your local database.
+		    <?php echo $LANG['BARCODE_INPUT_INSTRUCTIONS']; ?>
 		</div>
 		<div style="margin:20px 0px">
 			<hr/>
 		</div>
 		<div class="bottom-breathing-room">
-			<label for="occidsubmit">Occurrence ID:</label>
+			<label for="occidsubmit"><?php echo $LANG['OCCUR_ID']; ?>:</label>
```

Comment:
I think this was mislabeled by the original author of the code.
Please change "Occurrence ID" to "occid (Symbiota ID)"